### ISSUE TITLE
It does not work change the language.

### ISSUE THREAD

**Issue by andreili** [chocolatkey/FoOlSlide2#1]
Any change language interface displays only the English version.

**Comment by chocolatkey** [comment]
Hello, and sorry for the wait. Is this a fresh install? And you are talking about the "Default Software Language" correct?

**Comment by andreili** [comment]
Yes, it's a fresh install, from new database with create all tables on install.
Yes, it's talk the about "Default Software Languge".
Set to English: http://prntscr.com/9z7fzu
Set to Russian: http://prntscr.com/9z7g7d
After all changes clicked "Save" button.

PS: Sorry by my english, my level it's very low, don't used google translate :)

**Comment by chocolatkey** [comment]
Sorry for the wait. I managed to replicate the issue and have found that some other settings also fail to save to the database. I am currently trying to fix this and will tell you when I have!

**Comment by TIM04** [comment]
andreili, импортируй БД от старой с настройками и будет тебе счастье :)
